docs/troubleshooting.asciidoc

```diff
@@ -20,122 +20,105 @@ so that we can analyze the problem.
 Logs should include everything from when the application starts up until the first request executes.
 See <<debug-mode>> for more information.
 
-* <<no-data-sent>>
-* <<missing-performance-metrics>>
-* <<debug-mode>>
-* <<disable-agent>>
 
 [float]
-[[no-data-sent]]
-=== No data is sent to the APM Server
+[[use-latest-agent]]
+=== Updating to latest agent version
 
-If neither errors nor performance metrics are being sent to the APM Server,
-it's a good idea to first check your log file and look for output just as the app starts.
+The Elastic Node.js APM Agent is updated frequently and releases are not
+strongly tied to other components in the Elastic Stack.  Therefore, trying to
+update to the most recently released agent version is often the recommended
```

Comment:
```suggestion
strongly tied to other components in the Elastic Stack.  Therefore,
updating to the most recently released agent version is often the recommended
```

Reply:
> Is there a reason a user wouldn't be able to just update?

Probably not, no. :)  I had actually copied this language from https://www.elastic.co/guide/en/apm/agent/java/current/trouble-shooting.html#trouble-shooting-use-latest-agent

Eventually, when we have a new major version of the Node.js APM agent that drops support for some older versions of node, then the user might not be able to use the latest APM agent version. But that's a stretch. I agree the "trying to" doesn't need to be in there. Thanks.

Reply:
Damn. Missed opportunity to commit your suggestion with the commit message: "do or do not, there is no try"